onnxruntime/test/onnx/main.cc

```diff
@@ -739,6 +738,29 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
     broken_tests.insert({"nllloss_NCd1_weight_ii_expanded", "wait for investigation"});
     broken_tests.insert({"nllloss_NCd1d2_no_weight_reduction_mean_ii", "wait for investigation"});
     broken_tests.insert({"nllloss_NCd1d2_no_weight_reduction_mean_ii_expanded", "wait for investigation"});
+
+    // Disabled on error: shape mismatch, expect {} got {1}
+    // NNAPI does not support scalar output, all the outputs are tensors, scalar output will be a {1} tensor
+    {
+      broken_tests.insert({"sce_NCd1_mean_weight_negative_ii", "Shape mismatch"});
+      broken_tests.insert({"sce_NCd1_mean_weight_negative_ii_expanded", "Shape mismatch"});
```

Comment:
Do we have shape inferencing for the output that is expected to be {1} available when NNAPI is taking nodes? If so, can we modify the output shape from {} to {1} in the NNAPI EP?

Reply:
Since NNAPI does not really support {} tensor, for NNAPI EP we actually do not have shape inferencing to tell if an output should be {}, one way to do this can be when we build the NNAPI model, we can record all the graph output should be scalar, and we can replace {1} output in NNAPI model to {} output for NNAPI EP.

This is out of scope of this change, and I want to split the enable scalar output to a separated change, in case we want to include this one in 1.6

Reply:
Added map {1} tensor output back to scalar ({} tensor) if necessary
These model tests will pass